cloudmarker/test/data/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Use root/example as user/password credentials
+version: '3.1'
+
+services:
+
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: dummy
+      MONGO_INITDB_ROOT_PASSWORD: password
+    ports:
+      - 27017:27017
+    networks:
+      mynet:
+        ipv4_address: 172.16.101.2
+
+  mongo-express:
+    image: mongo-express
```

Comment:
I can see mongo-express is added only for a testing purpose of MongoDB store. I could not figure out any other use of this image. Is it okay to take this image into docker compose @susam as in future we need to maintain mongo-express separately only for testing the MongoDB store plugin.

Reply:
I am curious to know where this `mongo-express` image is being used.